### PR TITLE
@bentley/imodeljs-native 3.2.8

### PR DESCRIPTION
**Pull request by pmconne**

### CODE REVIEW

File: core/transformer/src/test/IModelTransformerUtils.ts [hunk 1]
```diff
@@ -507,8 +507,13 @@ export async function assertIdentityTransformation(
   expect(modelsOnlyInSourceAsInvariant).to.have.length(0);
   expect(onlyInTargetModels).to.have.length(0);
 
-  const makeRelationKey = (rel: any) => `${rel.SourceECInstanceId}\x00${rel.TargetECInstanceId}`;
-  const query: Parameters<IModelDb["query"]> = ["SELECT * FROM bis.ElementRefersToElements", undefined, { rowFormat: QueryRowFormat.UseECSqlPropertyNames }];
+  const makeRelationKey = (rel: any) =>
```
Comment (pmconne) [inline core/transformer/src/test/IModelTransformerUtils.ts:510]: @MichaelBelousov IModelTransformerUtils.ts contained 10 conflicts. Please verify I resolved them correctly.

Comment (MichaelBelousov) [reply core/transformer/src/test/IModelTransformerUtils.ts:510]: if the typescript builds (I didn't manually check that the imports are all correct), then `IModelTransformerUtil.ts` is correct

**Comment by pmconne** [comment]
@grigasp @saskliutas I removed references to non-existent presentation code snippets, please revisit in separate PR.
```
Could not find snippet 'Presentation.Content.Customization.PropertySpecification.IsReadOnly.Ruleset.' Requested by file 'presentation\content\PropertySpecification.md. Ensure it exists and is in D:\vsts_a\175\a\staging\extract\Presentation.Content.Customization.PropertySpecification.IsReadOnly.Ruleset

Could not find snippet 'Presentation.Content.Customization.PropertySpecification.IsReadOnly.Result.' Requested by file 'presentation\content\PropertySpecification.md. Ensure it exists and is in D:\vsts_a\175\a\staging\extract\Presentation.Content.Customization.PropertySpecification.IsReadOnly.Result

Could not find snippet 'Presentation.Content.Customization.PropertySpecification.Priority.Ruleset.' Requested by file 'presentation\content\PropertySpecification.md. Ensure it exists and is in D:\vsts_a\175\a\staging\extract\Presentation.Content.Customization.PropertySpecification.Priority.Ruleset
```

**Review by calebmshafer — APPROVED** [review]

**Comment by pmconne** [comment]
@MichaelBelousov is going to clean up flaky transformer test in separate PR. Appears to have a race condition on his state db.
Integration test failures are unrelated - occurring on all PRs.

**Comment by calebmshafer** [comment]
@pmconne sounds good, we'll follow up on the flaky integration tests tomorrow. @aruniverse it looks to be related to the new tests you added to the list :)

I think the flakiness is the reason they weren't being run in the first place. Although I agree we should just fix them rather than take them off the list again...

**Comment by grigasp** [comment]
> @grigasp @saskliutas I removed references to non-existent presentation code snippets, please revisit in separate PR.
> 
> ```
> Could not find snippet 'Presentation.Content.Customization.PropertySpecification.IsReadOnly.Ruleset.' Requested by file 'presentation\content\PropertySpecification.md. Ensure it exists and is in D:\vsts_a\175\a\staging\extract\Presentation.Content.Customization.PropertySpecification.IsReadOnly.Ruleset
> 
> Could not find snippet 'Presentation.Content.Customization.PropertySpecification.IsReadOnly.Result.' Requested by file 'presentation\content\PropertySpecification.md. Ensure it exists and is in D:\vsts_a\175\a\staging\extract\Presentation.Content.Customization.PropertySpecification.IsReadOnly.Result
> 
> Could not find snippet 'Presentation.Content.Customization.PropertySpecification.Priority.Ruleset.' Requested by file 'presentation\content\PropertySpecification.md. Ensure it exists and is in D:\vsts_a\175\a\staging\extract\Presentation.Content.Customization.PropertySpecification.Priority.Ruleset
> ```

@Alina657 please have a look

**Pull request merged**